ui/test/nuxeo-user-avatar.html

```diff
@@ -0,0 +1,74 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<html>
+  <head>
+    <title>nuxeo-user-avatar</title>
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes" />
+    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="../../../wct-browser-legacy/browser.js"></script>
+    <script src="../../../@nuxeo/nuxeo-elements/dedup.js"></script>
+    <script src="../../../@nuxeo/nuxeo-elements/test/test-helpers.js"></script>
+  </head>
+  <body>
+    <test-fixture id="nuxeo-user-avatar-fixture">
+      <template>
+        <nuxeo-user-avatar user>
+      </template>
+    </test-fixture>
+
+    <script type="module">
+      import { dom } from '@polymer/polymer/lib/legacy/polymer.dom.js';
+      import '@polymer/polymer/polymer-legacy.js';
+      import '../widgets/nuxeo-user-avatar.js';
+
+      suite('<nuxeo-user-avatar>', () => {
+        test('should get initial characters based on first and last name', () => {
+          const element = fixture('nuxeo-user-avatar-fixture');
+          element.user =
+          {
+            'entity-type': 'user',
+            id: 'jdoe',
+            properties: {
+              username: 'jdoe',
+              firstName: 'John',
+              lastName: 'Doe'
+            },
+          };
+          expect(dom(element.root).querySelector('#character').innerText).to.equal('JD');
+        });
+
+        test('should get the user icon if non Latin characters exist on user information', () => {
+          const element = fixture('nuxeo-user-avatar-fixture');
+          element.user =
+          {
+            'entity-type': 'user',
+            id: 'はると',
+            properties: {
+              username: 'はると',
+              firstName: 'はると',
+              lastName: 'りく'
+            },
+          };
+          return flush().then(() => {
+            const ironIcon = dom(element.root).querySelector('iron-icon');
+            expect(dom(element.root).querySelector('iron-icon').shadowRoot.hasChildNodes()).to.equal(true);
```

Comment:
Great to have tests, thanks!
Should you check the icon itself? Since we are using one specific icon, it may make sense to ensure that it is there and not a different one.